.github/workflows/ci_windows_x64_msvc.yml

```diff
@@ -45,8 +46,16 @@ jobs:
       - name: "Installing requirements"
         run: choco install ccache --yes
       - name: "Configuring MSVC"
-        uses: ilammy/msvc-dev-cmd@0b201ec74fa43914dc39ae48a89fd1d8cb592756 # v1.13.0
-      - name: "Building IREE"
-        run: ./build_tools/cmake/build_all.sh "${BUILD_DIR}"
+        uses: ilammy/msvc-dev-cmd@v1.13.0
+      - name: "Installing iree reqs"
+        run: |
+          choco install cmake
+          choco install ninja
+          Import-Module $env:ChocolateyInstall\helpers\chocolateyProfile.psm1
+          refreshenv
+      - name: "Building Iree"
+        run: bash ./build_tools/cmake/build_all.sh "/c/mnt/azure/build-windows"
       - name: "Testing IREE"
-        run: ./build_tools/cmake/ctest_all.sh "${BUILD_DIR}"
+        run: bash ./build_tools/cmake/ctest_all.sh "/c/mnt/azure/build-windows"
```

Comment:
I wan just hard coding it because the workflow goes between bash and powershell which have different notations for path.  I'll rewrite with 2 env variables

Reply:
Yeah, the env variable is being defined in bash and isnt carrying over to powershell

Reply:
Okay, writing out the path twice is fine for now.

We still shouldn't need the workflow itself to specify a functional path. That should be configured at the runner level.

With enough of these workarounds moved up into the runner config, we should be able to have the entire workflow use either powershell or bash as needed. We aren't going to have just one workflow - all of our workflows (like the Windows portions of https://github.com/iree-org/iree/blob/main/.github/workflows/pkgci_build_packages.yml, and workflows in other projects) need a reliable runner image to use, without this sort of patching.